skyhook.agent.nut

```diff
@@ -4,31 +4,27 @@
  *
  * @author Satya Yarramsetty <satya.yarramsetty@trueposition.com>
  *
- * @version 1.0.0
+ * @version 1.0.1
  */
 
 class skyhook {
-  static version = [1, 0, 0];
+  static version = [1, 0, 1];
 
   static REQUEST_TAG = "skyhook.wifiscan.request";
   static RESPONSE_TAG = "skyhook.wifiscan.response";
 
   static ERROR_NO_ACCESS_POINTS = "no wifi access points";
   static ERROR_UNSUCCESSFUL_RESPONSE = "unsuccessful response from Skyhook Precision Location server";
 
-  deviceName = null;
+  static SKYHOOK_LOCATION_URL = "https://api.skyhookwireless.com/wps2/location"; 
+
   skyhookKey = null
-  skyhookUrl = null;
 
   /**
-   * @param {string} devicename
    * @param {string} skyhookkey
-   * @param {string} skyhookurl
    */
-  constructor(devicename, skyhookkey, skyhookurl) {
```

Comment:
I guess since you are changing the APIs in an incompatible way, it should be a major version update, ei. "2.0.0"